core/src/main/resources/defaults/blueprints/cdp-sdx.bp

```diff
@@ -4,70 +4,128 @@
   },
   "description": "CDP 1.0 Shared Data Experience template with Atlas and Ranger",
   "blueprint": {
-    "cdhVersion": "7.0.0",
+    "cdhVersion": "6.0.99",
     "displayName": "datalake",
+    "hostTemplates": [
+      {
+        "cardinality": 1,
+        "refName": "master",
+        "roleConfigGroupsRefNames": [
+          "atlas-ATLAS_SERVER-BASE",
+          "hbase-GATEWAY-BASE",
+          "hbase-HBASETHRIFTSERVER-BASE",
+          "hbase-REGIONSERVER-BASE",
+          "hbase-MASTER-BASE",
+          "hdfs-DATANODE-BASE",
+          "hdfs-BALANCER-BASE",
+          "hdfs-GATEWAY-BASE",
+          "hdfs-NAMENODE-BASE",
+          "hdfs-NFSGATEWAY-BASE",
+          "hdfs-SECONDARYNAMENODE-BASE",
+          "hive-HIVEMETASTORE-BASE",
+          "kafka-GATEWAY-BASE",
+          "kafka-KAFKA_BROKER-BASE",
+          "knox-KNOX_GATEWAY-BASE",
+          "ranger-RANGER_ADMIN-BASE",
+          "ranger-RANGER_TAGSYNC-BASE",
+          "ranger-RANGER_USERSYNC-BASE",
+          "solr-SOLR_SERVER-BASE",
+          "zookeeper-SERVER-BASE"
+        ]
+      }
+    ],
     "services": [
       {
         "refName": "zookeeper",
-        "serviceType": "ZOOKEEPER",
         "roleConfigGroups": [
           {
+            "base": true,
             "refName": "zookeeper-SERVER-BASE",
-            "roleType": "SERVER",
-            "base": true
+            "roleType": "SERVER"
           }
-        ]
+        ],
+        "serviceType": "ZOOKEEPER"
       },
       {
         "refName": "ranger",
-        "serviceType": "RANGER",
+        "roleConfigGroups": [
+          {
+            "base": true,
+            "configs": [
+              {
+                "name": "process_auto_restart",
+                "value": "true"
```

Comment:
There should be a better way to enable process auto restart for all roles that support it.

Reply:
Thanks to Attila: #5140.

Reply:
Will remove the configurations for auto_restart as they will be injected by Cloudbreak.